store/sqlstore/post_store.go

```diff
@@ -280,12 +280,7 @@ func (s *SqlPostStore) Get(id string, skipFetchThreads bool) (*model.PostList, *
 	}
 
 	var post model.Post
-	var postFetchQuery string
-	if skipFetchThreads {
-		postFetchQuery = "SELECT p.*, (SELECT count(Posts.Id) FROM Posts WHERE Posts.RootId = p.Id) as ReplyCount  FROM Posts p WHERE p.Id = :Id AND p.DeleteAt = 0"
-	} else {
-		postFetchQuery = "SELECT * FROM Posts WHERE Id = :Id AND DeleteAt = 0"
-	}
+	postFetchQuery := "SELECT p.*, (SELECT count(Posts.Id) FROM Posts WHERE Posts.RootId = p.Id AND Posts.DeleteAt = 0) as ReplyCount FROM Posts p WHERE p.Id = :Id AND p.DeleteAt = 0"
```

Comment:
Can you elaborate why we do the `count` even when `!skipFetchThreads`? Presumably the caller doesn't need it at that point, right?

Reply:
@lieut-data - there are several code paths leading to post_store from APIs regarding Flagged, Pinned and Search posts which needed reply-count fields.
I had two options:
1. Modify all APIs and pass fetchThreads like I've done with Get/GetAround (this would require changing redux and webapp as well)
2. Get the counts regardless. If webapp is a new one - it'll take reply counts and use them, the old webapp will ignore it.
There is a slight performance impact on Old-Webapp-New-Server combo, but I think it's an obscure case and can be ignored.